spark/src/main/scala/org/apache/spark/sql/delta/DeltaOperations.scala

```diff
@@ -77,7 +78,8 @@ object DeltaOperations {
 
   abstract class OperationWithPredicates(name: String, val predicates: Seq[Expression])
       extends Operation(name) {
-    private val predicateString = JsonUtils.toJson(predicatesToString(predicates))
+    private val predicateString = JsonUtils.toJson(predicatesToString(predicates).map(
```

Comment:
256 seems like a very small limit?

Could there be cases where users are relying on the output of DESCRIBE HISTORY (where these predicates do show up) and have some business logic on top of it? There business logic may break if predicates are stripped at such a low threshold.

Reply:
could be possible . 
any suggestion for limit ?

Reply:
How about 4096? That still seems humanly parseable. The real problem is when these do get very big, i.e. the string is multiple megabytes large

Reply:
In general the value must be configurable using a flag such that users can lift that limit if they require